tests/test_file_filter.py

```diff
@@ -87,6 +87,42 @@ def test_get_file_filter_terms():
     assert filter_terms == ["file 1\\.org", "/path/to/dir/.*\\.org"]
 
 
+def test_exclude_file_filter_terms():
+    # Arrange
+    file_filter = FileFilter()
+    q_with_filter_terms = 'head tail file:-"file 1.org" file:"/path/to/dir/*.org"'
```

Comment:
Shouldn't the tested query be something like below?
```suggestion
    q_with_filter_terms = 'head tail -file:"file 1.org" file:"/path/to/dir/*.org"'
```